verifier/trustpolicy/trustpolicy.go

```diff
@@ -136,185 +134,46 @@ var (
 
 var supportedPolicyVersions = []string{"1.0"}
```

Comment:
nit:
`supportedPolicyVersions` might not be a shared variable between `blob` and `oci` trust policy documents in the future. Their versions may develop separately.

Reply:
Yep I agree and that's the reason its private variable. When it diverges we will create new ones variables and validations.